packages/common/src/ui/components/popover/BasePopover/usePopover.tsx

```diff
@@ -50,8 +50,8 @@ export const usePopover = (): UsePopoverControl => {
         left: e.clientX,
         bottom: e.clientY,
         right: e.clientX,
-        width: 0,
-        height: 0,
+        width: 25,
+        height: 25,
```

Comment:
this is making the virtual ref has some width/height which seems to help. Alternative I tried playing with was to try offset the left/bottom/right/top a little based on the position of the popover.. this worked a little better